src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="#">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">      
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img class="product-card__img" src="images/imac.jpeg" alt="Imac">
+
+      <h4 class="product-card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h4>
+
+      <p class="product-card__code">
+        Product code: 195434
+      </p>
+
+      <div class="product-card__review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="product-card__reviews">
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="product-card__price">
+        <div class="product-card__price-title">
+          Price:
+        </div>
+
+        <div class="product-card__price-count">
+          $2,199
+        </div>
+      </div>
+
+      <a class="product-card__button" data-qa="hover">
+        buy
+    </a>
```

Comment:
```suggestion
      <a class="product-card__button" data-qa="hover">
        buy
      </a>
```
Closing tag should be under the opening